bin/samples/server/petstore/java-play-framework-controller-only/.swagger-codegen-ignore

```diff
@@ -0,0 +1,23 @@
+# Swagger Codegen Ignore
```

Comment:
This complete commit is wrong. The samples already exist in "samples", not in "bin/samples". You should revert it and generate is from the right place. Thanks!